dll/win32/cryptui/main.c

```diff
@@ -3717,7 +3731,9 @@ static UINT CALLBACK cert_properties_general_callback(HWND hwnd, UINT msg,
         data = (struct edit_cert_data *)GetWindowLongPtrW(hwnd, DWLP_USER);
         if (data)
         {
+#ifdef __REACTOS__
```

Comment:
ifdef REACTOS? So does this mean that this code is not in wine?

Reply:
This code is not in wine.

Reply:
No, it is:
https://source.winehq.org/git/wine.git/blob/8c74027ba60a360f24cf232ee233e71262d8f1ff:/dlls/cryptui/main.c#l3720

Reply:
Sorry. Fixed.